agent/api/task.go

```diff
@@ -202,23 +208,64 @@ func (task *Task) initializeCredentialsEndpoint(credentialsManager credentials.M
 
 }
 
+// BuildCNIConfig constructs the cni configuration from eni
+func (task *Task) BuildCNIConfig() (*ecscni.Config, error) {
+	if !task.isNetworkModeVPC() {
+		return nil, errors.New("task isnot enabled with vpc networkmode")
```

Comment:
spaces, also context: `"task config: task has no ENIs associated with it, unable to generate cni config"`